pkg/actors/actors.go

```diff
@@ -239,7 +239,7 @@ func newActorsWithClock(opts ActorsOpts, clock clock.WithTicker) (ActorRuntime,
 		if opts.Config.SchedulerClients == nil {
 			return nil, fmt.Errorf("scheduler reminders are enabled, but no Scheduler clients are available")
 		}
-		log.Info("Using Scheduler service for reminders.")
+		log.Debug("Using Scheduler service for reminders.")
```

Comment:
Here - Before merging I want to make sure it doesn't break the test, or can we add the mode to the int test that was ensuring this log line if `SchedulerReminders` is enabled?

Reply:
@cicoyle done

Reply:
thank you!